code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -119,9 +119,9 @@
 	to_chat(src, "<b>Clicking on any object will try to consume it, either deconstructing it into its components, destroying it, or integrating any materials it has into you if successful.</b>")
 	to_chat(src, "<b>Ctrl-Clicking on a mob will attempt to remove it from the area and place it in a safe environment for storage.</b>")
 	to_chat(src, "<b>Prime Directives:</b>")
-	to_chat(src, "1. Consume resources and replicate until there are no more resources left.")
-	to_chat(src, "2. Ensure that the station is fit for invasion at a later date, do not perform actions that would render it dangerous or inhospitable.")
-	to_chat(src, "3. Biological and Sentient resources will be harvested at a later date, do not harm them.")
+	to_chat(src, "1. <span class='warning'>Consume resources and replicate until there are no more resources left.</span>")
+	to_chat(src, "2. <span class='warning'>Ensure that the station is fit for invasion at a later date, do not perform actions that would render it dangerous or inhospitable.</span>")
+	to_chat(src, "3. <span class='warning'>Biological and Sentient resources will be harvested at a later date, do not harm them and do not actively seek to confront them save for defense.</span>")
```

Comment:
Might want to make this 'your self defense' rather than just 'defense', as the latter could imply 'defense of the station'.

Reply:
true, any other ideas on how to word it?

Reply:
I really would go with 'your self defense'. "Do not actively seek to confront them. Defending yourself from direct attack is still permitted" or similar would work too.

Reply:
Self Defense implies they are not allowed to push into new areas once they have exhausted resources in other areas.

Reply:
No, it does not imply that. The 'your self defense' part only applies to law 3. It does not limit law 1.

Reply:
@Kyep Yes, because the way silicon laws are enforced, this will inevitably lead to someone getting antag banned or server banned for attacking a crew member who happens be guarding resources caches.

Reply:
I don't know why half the arguments on this are paranoia about admins wanting to ban swarmer players. Silicon laws are absolutely not that strictly micromanaged at all. Since when do admins intentionally bait players into getting banned?? Enough with the persecution complex...

Reply:
I'm more confused with the sudden desire to make antag roles less antagonizing and safe space them. These 'fixes' don't improve the server at all, especially compared to things like vampire that are need of overhauls and fixing of various abuses of their abilities.

Reply:
Vampires are off-topic for this PR. Yes, law 1 overrides law 3, and this could be used to justify doing pretty much anything in the name of resource gathering. Only swapping the positions of laws 1 and 3 would really solve this. Even then, you'd need to alert 3 (now 1) to make it clear that self defense is OK, but 'defending the station' (ie: attacking blobs, nuke ops, etc) is not.